client/views/actions/channel_list.tpl

```diff
@@ -0,0 +1,18 @@
+<table style="width:100%">
```

Comment:
I think we should avoid introducing inline styles like this in our templates. Can this be set in the CSS instead?

Also, how do you trigger this exactly? I tried `/list` but no luck.
Could you add a screenshot to the PR description please? :-)